doc/sphinx/source/recipes/recipe_perfmetrics.rst

```diff
@@ -61,6 +61,35 @@ User settings in recipe
    * zonal_ymin: for plot_type zonal only, minimum pressure level on the y-axis (default: 5. hPa)
    * latlon_cmap: for plot_type latlon only, chosen color table (default: "amwg_blueyellowred")
    * plot_units: plotting units (if different from standard CMOR units)
+   * add_tropopause: adds an outline of a climatological tropopause to the zonal plot (default: False)
+
+   *Special optional plot configurations*
+
+     It is possible to make some specific customizations to the plots (zonal
+     only).
+
+     This includes for example specific tickmark labels of the axes.
+
+     Those special customizations can be done by adding ncl plotting resources
+     combined with prefix ``res_`` as optional settings of the main script in the
+     recipe.
+
+     Note that this requires to be familiar with the ncl plotting routines for
+     pressure vs height plots
+     (https://www.ncl.ucar.edu/Document/Graphics/Interfaces/gsn_csm_pres_hgt.shtml)
+     and the corresponding resources.
+
+     The following shows an example on customizing the latitude tickmarks so
+     that a degree sign and and empty space is used for the labels:
+
+  .. code-block:: yaml
```

Comment:
It looks like there might be something wrong with the indentation. When you look at the .rst file (https://github.com/ESMValGroup/ESMValTool/blob/add_tropopause/doc/sphinx/source/recipes/recipe_perfmetrics.rst), the paragraph following this code-block ("Required settings (variables)") appears as if it would be part of the code-block.

Reply:
The indentation is now corrected.